src/osp/OSPApplication.h

```diff
@@ -61,10 +63,14 @@ class OSPApplication
     size_t debug_num_packages() const { return m_packages.size(); }
 
     universe::Universe& get_universe() { return m_universe; }
+    
+    std::shared_ptr<spdlog::logger> get_logger() { return m_logger; };
```

Comment:
I wonder if it's worth returning a raw pointer or a reference or something, to prevent incrementing the `std::shared_ptr` ref counter, @jonesmz any thoughts?

Reply:
@EhWhoAmI @z-adams you should return a const& to the m_logger variable.
Be wide in what you accept, and narrow in what you return.

@Capital-Asterisk that's how spdlog works, basically.